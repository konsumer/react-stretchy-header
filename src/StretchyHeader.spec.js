import initStoryshots from '@storybook/addon-storyshots'

global.requestAnimationFrame = (callback) => {
  setTimeout(callback, 0)
}

initStoryshots()
