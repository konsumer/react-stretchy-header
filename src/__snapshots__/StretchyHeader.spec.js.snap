// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots StretchyHeader usage 1`] = `
<div>
  STRETCHY
</div>
`;
